build/publish/AzureInfo.targets

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Condition="'$(GeneratePropsFile)' != 'true'"  Project="$(NuGetPackagesDir)/microsoft.dotnet.build.tasks.feed/$(BuildTasksFeedToolVersion)/build/Microsoft.DotNet.Build.Tasks.Feed.targets" />
+          
+  <Target Name="SetAzureProps"
+          Condition=" '$(IsOrchestratedPublish)' == 'True' And '$(PB_PublishBlobFeedUrl)' != '' ">
+
+    <ParseBlobUrl BlobUrl="$(PB_PublishBlobFeedUrl)">
+      <Output TaskParameter="BlobElements" ItemName="BlobElements" />
+    </ParseBlobUrl>
+
+    <PropertyGroup>
+      <ArtifactContainerName>%(BlobElements.ContainerName)</ArtifactContainerName>
+      <ChecksumContainerName>%(BlobElements.ContainerName)</ChecksumContainerName>
+      <ArtifactCloudDropAccountName>%(BlobElements.AccountName)</ArtifactCloudDropAccountName>
+      <ChecksumCloudDropAccountName>%(BlobElements.AccountName)</ChecksumCloudDropAccountName>
+      <BlobStoragePartialRelativePath>%(BlobElements.BlobPath)/assets</BlobStoragePartialRelativePath>
```

Comment:
For the assets bits, you'll need to strip off the /index.json first, either before parsing or at this point.

Reply:
@mmitche 
So you are saying that the "BlobPath" = '20180110-13/cli/index.json' - then what is the 'EndPoint'?

Reply:
Yes, PB_PublishBlobFeedUrl contains the index.json.  When you parse the blob url, BlobPath would be 20180110-13/cli/index.json.  Therefore, appending /assets to the end would be 20180110-13/cli/index.json/assets, which would be invalid (index.json is a file)